### DISCUSSION TITLE
Sources in RVX Manager?

### DISCUSSION THREAD

**Discussion by ditchmagnet** [anddea/revanced-patches#215] [Q&A] [answered]
I had been using default sources and wondering if I need to change something else in rvx manager to use these patches?

**Comment by anddea** [comment, marked as answer]
Change organization to `anddea`.

Patches
```
anddea
```

```
revanced-patches
```

Integrations
```
anddea
```
```
revanced-integrations
```

The organization is always the name of the author, and patches/integrations are the names of the author's repositories.

**Reply by BLACK4585** [reply]
Just did so, but in the patcher it shows `Recommended: v19.05.36`. Do I have to change the API URL too?

**Reply by anddea** [reply]
> it shows `Recommended: v19.05.36

It shows it because one of the patches is not available for newer versions, not available because it's no longer relevant and the patch is obsolete. 

You don't have to change anything else.

**Reply by BLACK4585** [reply]
So I can just go with the newest YT Version? Or what's the currently supported, newest version?
Also, a question apart from that, what is the difference between your patches and the ones by rufusin? I saw you already merged sth a while ago…

**Reply by anddea** [reply]
Latest is 19.16. rufusin bumps supported app version and fixes bugs that lateer will be here as well.
